src/main/java/org/mule/module/apikit/ApikitExtensionLoadingDelegate.java

```diff
@@ -116,6 +117,10 @@ public void accept(ExtensionDeclarer extensionDeclarer, ExtensionLoadingContext
         .withDsl(ParameterDslConfiguration.builder().allowsReferences(false).build())
         .withExpressionSupport(NOT_SUPPORTED)
         .ofType(typeBuilder.arrayType().of(typeLoader.load(FlowMapping.class)).build());
+    parameterGroupDeclarer.withOptionalParameter("attributesDeserializingStrategies")
+        .withDsl(ParameterDslConfiguration.builder().allowsReferences(false).build())
+        .withExpressionSupport(NOT_SUPPORTED)
+        .ofType(typeLoader.load(AttributesDeserializingStrategies.class));
```

Comment:
AttributesDeserializingStrategies enum parameter is not necessary? please check only COMMA can be set

Reply:
The type wraps the enum so there is no need to add additional setup for the parameter, it is being validated.